examples/systemcalls/systemcalls.c

```diff
@@ -17,6 +18,12 @@ bool do_system(const char *cmd)
  *   or false() if it returned a failure
 */
 
+int res;
+	res= system(cmd); 
+	
+	if(res == -1)
```

Comment:
@chaisuresh think about what would happen if the cmd returned with an error code other than -1 here.